### PR TITLE
Improve trades.get_any performance

### PR DESCRIPTION
**Pull request by FelixKleinpeter**
The function has changed a lot and looks more complicated and has different arguments now, but it is very faster for big amounts of datas, so its possible to run it on rokko. Please tell me if I should change anything. I use it for the experiments, maybe we can think about a definitive version of it.

### CODE REVIEW

**Comment by keisuke-umezawa** [comment]
@FelixKleinpeter Can I review it? you already finished implementation?

**Comment by FelixKleinpeter** [comment]
I will finish to set it usable for deployment. At the moment I focused on increasing the speed. It will be done when you will come back from your holidays!

**Comment by FelixKleinpeter** [comment]
I fixed types and add a new function. The code is more complicated but the new function makes possible to run on rokko. Thanks to apply your review!

File: src/backlight/strategies/filter.py [hunk 1]
```diff
@@ -57,3 +59,46 @@ def skip_entry_by_spread(
             deleted_ids.append(i)
 
     return trades[~trades["_id"].isin(deleted_ids)]
+
+
+def get_in_set(trades: Trades, unit: str, container_set: tuple) -> Type["Trades"]:
```
Comment (keisuke-umezawa) [inline src/backlight/strategies/filter.py]: get_in_set -> filter_entry_by_time?

**Comment by FelixKleinpeter** [comment]
Last changes are done. Can I merge it?

**Pull request merged**